connectivity/lorawan/mbed_lib.json

```diff
@@ -5,6 +5,9 @@
             "help": "LoRa PHY region: EU868, AS923, AU915, CN470, CN779, EU433, IN865, KR920, US915",
             "value": "EU868"
         },
+        "radio": {
+            "help": "value set in radio driver : SX126X, SX1272, SX1276"
```

Comment:
What happen if a value is provided in  `mbed_app.json` ?

Reply:
It will not be used as define will be MBED_CONF_APP_xxx